### ISSUE TITLE
NO delay

### ISSUE THREAD

**Issue by TDMidkwhatisxd** [montlikadani/SetTheSpawn#12]
I set the config to enable the bar, and set 5 seconds delay, i get insta spawned :/

**Comment by montlikadani** [comment]
Because you not reloaded the plugin

**Comment by TDMidkwhatisxd** [comment]
I just did

**Comment by TDMidkwhatisxd** [comment]
restarted server too

**Comment by montlikadani** [comment]
What version of SetTheSpawn you using? (not latest)

**Comment by TDMidkwhatisxd** [comment]
2.6.5

**Comment by TDMidkwhatisxd** [comment]
1.16.3 papermc

**Comment by montlikadani** [comment]
If you're op then make sure that you denied the `setthespawn.bypass.cooldown` permission on your group.

**Comment by TDMidkwhatisxd** [comment]
let me try again with my alt

**Comment by TDMidkwhatisxd** [comment]
Ya works, thanks. :3

**Comment by MindofMars** [comment]
How denied the setthespawn.bypass.cooldown ?

**Comment by montlikadani** [comment]
With permission plugin
